Cargo.toml

```diff
@@ -20,6 +20,7 @@ members = [
 ]
 exclude = [
   "eng/scripts",
+  "target",
```

Comment:
This is always ignored. Why did you add it? This is going to raise a lot of eyebrows.

Reply:
it's not ignored 😬.  The targets folder will get package folder in it.  If you run any `cargo` commands on the packages in /targets, it complains about the package thinking its running in a workspace but not being a member or excluded

Reply:
Why are you running commands against anything `targets`? That's not our place space. For package analysis - though I maintain there's nothing to analyze (it's zipped source) - that should probably be done outside the source repo in the temp working dir AzDO, GHA, etc. give us. Let's not unnecessarily complicate things.